### PR TITLE
Send an email if the address is already bound to an user account

### PR DESCRIPTION
**Pull request by mcalinghee**
### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://element-hq.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [x] [Code style](https://element-hq.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://element-hq.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

**Comment by MatMaul** [comment]
Outside of CLA signing it seems ready.

I am moving that out of draft for early review, we are waiting for the legal department to clear the CLA.

Thanks!

**Comment by MatMaul** [comment]
AH, I can't move this from draft to ready to review, only people with write access can do this, this feels... weird?

I am not sure if this is a setting or GH default that we can't change.

@erikjohnston @reivilibre little ping on this since this PR will probably not be looked at until it's out of draft. Thanks !

**Comment by reivilibre** [comment]
> AH, I can't move this from draft to ready to review, only people with write access can do this, this feels... weird?

it's because it's not your PR, right? :-)

**Comment by MatMaul** [comment]
🤦 indeed probably, thanks for doing it.

File: synapse/res/templates/already_in_use.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% extends "_base.html" %}
+{% block title %}Email already in use{% endblock %}
+
+{% block body %}
+<p>You have asked us to register this email with a new Matrix account, but this email is already registered with an existing account.</p>
+
+<p></p>Please reset your password if needed.</p>
```
Comment (reivilibre) [inline synapse/res/templates/already_in_use.html:7]:
```suggestion
<p>Please reset your password if needed.</p>
```

**Review by reivilibre — APPROVED** [review]

**Comment by MatMaul** [comment]
@reivilibre any chance we get that merge ? Thanks !

**Comment by reivilibre** [comment]
@MatMaul  @mcalinghee please can you push a commit to the branch to kick CI or something? I don't have the access to your branch to do that :)

**Pull request merged**